interop-testing/src/main/java/io/grpc/testing/integration/Http2Client.java

```diff
@@ -272,6 +272,7 @@ private void rstDuringData() throws Exception {
 
     private void goAway() throws Exception {
       assertResponseEquals(blockingStub.unaryCall(simpleRequest), goldenResponse);
+      TimeUnit.SECONDS.sleep(1);
```

Comment:
I'm not clear about how the test is working. Where is verification that a GOAWAY was sent/seen?